freespace2/SDLGraphicsOperations.cpp

```diff
@@ -153,8 +155,10 @@ SDLGraphicsOperations::SDLGraphicsOperations() {
 }
 SDLGraphicsOperations::~SDLGraphicsOperations() {
 	SDL_QuitSubSystem(SDL_INIT_VIDEO);
-	ImGui_ImplSDL2_Shutdown();
-	ImGui_ImplOpenGL3_Shutdown();
+	if (!Cmdline_vulkan) {
+		ImGui_ImplSDL2_Shutdown();
```

Comment:
In the manager you run the SDL2 Frame in Vulkan mode, but here you don't do the shutdown in Vulkan mode.
These should likely be symmetric, so either don't do the SDL2 Frame in Vulkan mode, or (more likely AFAICT) do the SDL2 shutdown in vulkan mode as well